ansible/templates/consul-acl.hcl.j2

```diff
@@ -1,10 +1,10 @@
 {% raw -%}
 acl {
-  default_policy = "deny"
   down_policy = "extend-cache"
   enabled = true
   tokens {
     master = "master"
   }
+  {% endraw %}
+  default_policy = "{{ lookup('env', 'consul_acl_default_policy') }}"
```

Comment:
Just out of curiosity, is this after the raw section because it is jinja2-templating?

Reply:
Yep